src/utils/db.js

```diff
@@ -0,0 +1,9 @@
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  dialect: 'postgres',
+});
```

Comment:
Ensure that all database-related environment variables (`DB_HOST`, `DB_USER`, `DB_PASSWORD`, `DB_DATABASE`) are correctly set. Missing or incorrect values will prevent the database connection from being established.